CONTRIBUTING.md

```diff
@@ -50,14 +50,41 @@ These instructions cover new releases of the blocks plugin for those with commit
 
 ### Prerequisites
 
-- Have [Github's Hub](https://github.com/github/hub) cli installed and authenticated.
+-   Have [Github's Hub](https://github.com/github/hub) cli installed and authenticated.
 
 **Before any release** ensure you update:
 
-- The `readme.txt` file supported versions, changelog and list of blocks if the release includes new blocks.
+-   The `readme.txt` file supported versions, changelog and list of blocks if the release includes new blocks.
 
 > Note: version numbers will automatically be updated in files via the deploy script (see `bin/version-changes.sh`).
 
+### Creating release branches
+
+When releasing a major or minor feature frozen version of Blocks (e.g. release candidate), create a new release branch from master named `release/X.X`. **This release branch is where you should create tags and releases from.**
+
+If post-release fixes or changes are needed, create Pull Requests against **master**, but label the PR with the correct milestone. For example, if fixing an issue in Blocks 2.5, create a PR against master with a `2.5.x` milestone. Once approved and merged to master the same changes will be cherry picked to the release branch.
+
+To do this:
+
+-   Switch to the release branch
+-   Run the command ([using HUB command line helper](https://hub.github.com/hub-am.1.html)): `hub am -3 http://URL-TO-PR`
+-   The command will tell you if the merge was successful. If not, you can cherry pick individual commits using [git cherry pick](https://git-scm.com/docs/git-cherry-pick) which is more time consuming.
+-   Push the changes to origin.
+-   After success, label the PR as `status: cherry-picked`. This lets everyone know the chery picking has been completed.
+
+You'll also need to manually update the changelog and version numbers in the release branch before tagging. There is a changelog helper script which can list changelog entries from merged PRs in a milestone:
+
+`npm run changelog`
+
+Follow the instructions, then add the changelog entries to the readme.txt file before committing to the release branch. As well as the changelog, there are version strings which need updating in:
+
+-   `package.json`
+-   `readme.text`
+-   `src/package.php`
+-   `woocommerce-gutenberg-products-block.php`
```

Comment:
Version string updates should automatically get taken care of by the `version-changes.sh` script as part of the deploy script.